### PR TITLE
[DOCS] <new link for the custom profile link of the user>

### LINKED ISSUE #6216: [DOCS] <new link for the custom profile link of the user>
### Description

Under the [QuickStart Documentation](https://linkfree.io/docs/quickstart)
For point no 19. It refers to the old URL used and not the new one.

> Once merged your profile will be available a few minutes later on your custom url ([linkfree.eddiehub.io/SaraJaoude](https://linkfree.eddiehub.io/SaraJaoude))

Actually, it should be like this

> Once merged your profile will be available a few minutes later on your custom url ([linkfree.io/SaraJaoude](https://linkfree.io/SaraJaoude))

### Screenshots

Screenshot 2023-04-19 at 4 24 35 PM

### Additional information

I will be happy if this issue is assigned to me. I can fix and create the related PR for this.

### PR DESCRIPTION
**Pull request by munnokd**
## Fixes Issue #6216
Closes #6216

## Changes proposed
I changed url ([linkfree.eddiehub.io/SaraJaoude](https://linkfree.eddiehub.io/SaraJaoude)) to url ([linkfree.io/SaraJaoude](https://linkfree.io/SaraJaoude)) in the [QuickStart Documentation](https://linkfree.io/docs/quickstart)

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6339"></a>

### CODE REVIEW

File: pages/docs/quickstart.mdx [hunk 1]
```diff
@@ -145,7 +145,7 @@ An approving review from one of LinkFree's maintainers will show a green check m
 ![maintainer review](https://user-images.githubusercontent.com/35927536/217060602-ba3678d1-f655-4805-a47d-e40f1ddcd59f.png)
 
 18. You will receive a GitHub notification when you have a comment, review or your Pull Request has been merged
-19. Once merged your profile will be available a few minutes later on your custom url (<Link href="https://linkfree.eddiehub.io/SaraJaoude">linkfree.eddiehub.io/SaraJaoude</Link>)
+19. Once merged your profile will be available a few minutes later on your custom url (<Link href="https://linkfree.io/SaraJaoude">linkfree.io/SaraJaoude</Link>)
```
Comment (CBID2) [inline pages/docs/quickstart.mdx:148]:
```suggestion
19. Once merged, your profile will be available a few minutes later on your custom URL (<Link href="https://linkfree.io/SaraJaoude">linkfree.io/SaraJaoude</Link>)
```

File: pages/docs/quickstart.mdx [hunk 2]
```diff
@@ -145,7 +145,7 @@ An approving review from one of LinkFree's maintainers will show a green check m
 ![maintainer review](https://user-images.githubusercontent.com/35927536/217060602-ba3678d1-f655-4805-a47d-e40f1ddcd59f.png)
 
 18. You will receive a GitHub notification when you have a comment, review or your Pull Request has been merged
```
Comment (CBID2) [inline pages/docs/quickstart.mdx:147]:
```suggestion

18. You will receive a GitHub notification when you have a comment, review, or your Pull Request has been merged. 
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @munnokd! 😄 I just made some suggestions on how to make the instructions clearer. I hope you find them helpful! 😄

**Comment by munnokd** [comment]
Okay  @CBID2 I will change it.

**Comment by munnokd** [comment]
Hey, @CBID2 I completed the changes given by you. Please review my PR.

**Review by CBID2 — APPROVED** [review]
Looks great @munnokd! 😊 LGTM! 😊

**Review by eddiejaoude — DISMISSED** [review]
Thank you 👍

File: pages/docs/quickstart.mdx (hunk 2 above)
Comment (eddiejaoude) [reply pages/docs/quickstart.mdx]: No other bullet points end in a `.`, so adding one here wouldn't be consistent

File: pages/docs/quickstart.mdx [hunk 3]
```diff
@@ -144,8 +144,8 @@ An approving review from one of LinkFree's maintainers will show a green check m
 
 ![maintainer review](https://user-images.githubusercontent.com/35927536/217060602-ba3678d1-f655-4805-a47d-e40f1ddcd59f.png)
 
-18. You will receive a GitHub notification when you have a comment, review or your Pull Request has been merged
-19. Once merged your profile will be available a few minutes later on your custom url (<Link href="https://linkfree.eddiehub.io/SaraJaoude">linkfree.eddiehub.io/SaraJaoude</Link>)
+18. You will receive a GitHub notification when you have a comment, review, or your Pull Request has been merged.
```
Comment (eddiejaoude) [inline pages/docs/quickstart.mdx:147]:
```suggestion
18. You will receive a GitHub notification when you have a comment, review, or your Pull Request has been merged
```

**Pull request merged**